### ISSUE TITLE
less does not work

### ISSUE THREAD

**Issue by mitjade** [sysgears/apollo-universal-starter-kit#386]
After latest changes, I get errors if trying replace `sass` with `less`.

Pushed a branch with minimal changes needed to reporduce - https://github.com/sysgears/apollo-universal-starter-kit/tree/less

Did update to `node` `v8.5.0` and `yarn` `v1.1.0`, if this makes any difference?

Stack Trace:
```
Main.jsx:98 Uncaught TypeError: __WEBPACK_IMPORTED_MODULE_13_react_ga___default.a.initialize is not a function
    at Object../src/client/app/Main.jsx (Main.jsx:98)
    at __webpack_require__ (bootstrap 4d2f91c87fdcb5cf5b41:677)
    at fn (bootstrap 4d2f91c87fdcb5cf5b41:87)
    at Object../src/client/index.jsx (index.jsx:1)
    at __webpack_require__ (bootstrap 4d2f91c87fdcb5cf5b41:677)
    at fn (bootstrap 4d2f91c87fdcb5cf5b41:87)
    at Object.0 (log.js:18)
    at __webpack_require__ (bootstrap 4d2f91c87fdcb5cf5b41:677)
    at ./app.json.module.exports.app.debugSQL (bootstrap 4d2f91c87fdcb5cf5b41:723)
    at index.4d2f91c87fdcb5cf5b41.js:727
```

**Comment by vlasenko** [comment]
@mitjade This looks like as a problem with calling `react-ga` from `Main.jsx`

**Comment by mitjade** [comment]
@vlasenko I reverted to before I merged latest changes and I still get this? Before did not!

**Comment by mitjade** [comment]
But it works if I try on master.

**Comment by mitjade** [comment]
Locking `react-ga` to `v2.2.0` seems to fix the problem, still not sure why this only happens when I add `less`?

Will close for now.

**Comment by vlasenko** [comment]
@mitjade Maybe because you have regenerated `yarn.lock` and buggy version of `react-ga` has been pulled in

**Comment by mitjade** [comment]
@vlasenko Very likely. Should we lock this version in master, to avoid this happening to others?

**Comment by vlasenko** [comment]
@mitjade Lets think about it, when someone else reports an issue

**Comment by mitjade** [comment]
@vlasenko Ok. It might get resolved soon anyway.

**Comment by vlasenko** [comment]
@mitjade Yep, I saw they have published two versions today, 2.3.0 and 2.3.1. So something is happening with this package today

**Comment by vlasenko** [comment]
@mitjade Because in master's `yarn.lock` it uses `2.2.0` and in your branch's `yarn.lock` it uses `2.3.1`. Check `yarn.lock` contents in both branches and you will see difference
